packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -635,6 +636,83 @@ function pushStringAttribute(
   }
 }
 
+// Since this will likely be repeated a lot in the HTML, we use a more concise message
```

Comment:
could make a $RF helper for this?

Reply:
oh you mean to call a global. Yea, but this might also be opened in new tabs etc. which is annoying.

Reply:
I suppose it doesn't matter much if the error message isn't great for that case.

Reply:
yeah it's fine